lib/_http_server.js

```diff
@@ -490,6 +490,10 @@ Server.prototype.closeIdleConnections = function() {
   const connections = this[kConnections].idle();
 
   for (let i = 0, l = connections.length; i < l; i++) {
+    if (connections[i].socket._httpMessage && !connections[i].socket._httpMessage.finished) {
```

Comment:
I am worried this addition will make https://github.com/nodejs/node/issues/43795 worse.

Reply:
Well, we gotta fix that test (next on my list for Monday), but the server current does not behaves as the documentations says (and the docs reflects the original intended behavior) so I think we should fix this.